src/webgpu/shader/validation/statement/break_if.spec.ts

```diff
@@ -0,0 +1,49 @@
+export const description = `Validation tests for 'break if' statements'`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { scalarTypeOf, Type } from '../../../util/conversion.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kTestTypes = [
+  'bool',
+  'i32',
+  'u32',
+  'f32',
+  'f16',
+  'vec2f',
+  'vec3h',
+  'vec4u',
+  'vec3b',
+  'mat2x3f',
+  'mat4x2h',
```

Comment:
This removed a bunch of the extra testing for matrices and vectors, which I think is fine, but it also dropped Struct which I think we possibly want to keep. We may also want to add array, texture, atomic and sampler here as well?

Reply:
User-declared and non-constructable types require a bit of work with this pattern. I agree, but I'd like to do this for all the statement tests in this directory, as they all follow the same pattern.

Reply:
K, can you please file a bug with the details so we don't lose track?

Reply:
Done: https://github.com/gpuweb/cts/issues/3685